http/logical.go

```diff
@@ -185,16 +185,9 @@ func handleLogicalInternal(core *vault.Core, injectDataIntoTopLevel bool) http.H
 				}
 			}
 			switch req.Path {
+			// Route the token wrapping request to its respective sys NS
```

Comment:
Another potential issue with logging here  (and potentially returning early if validation fails) instead of at the logical layer is that we won't audit log the response.

Reply:
It's been a while, but I think we settled on keeping things as-is.